modoboa_pdfcredentials/documents.py

```diff
@@ -44,37 +56,82 @@ def page_template(canvas, doc):
         canvas.setCreator("Modoboa")
         footer = [Paragraph(_("Powered by Modoboa - Mail hosting made simple"),
                             styles["Footer"])]
-        Frame(0, 0, 21 * cm, 4 * cm).addFromList(footer, canvas)
+        Frame(0, 0, 21 * cm, 2 * cm).addFromList(footer, canvas)
 
     filename = lib.get_creds_filename(account)
     buff = BytesIO()
     doc = SimpleDocTemplate(buff, pagesize=A4)
+
     story = []
-    story.append(resized_image(lib.get_document_logo(), 6*cm))
+
+    story.append(resized_image(lib.get_document_logo(), 8*cm))
+
     story.append(Spacer(1, 1 * cm))
     story.append(Paragraph(_("Personal account information"), styles["Title"]))
-    story.append(Spacer(1, 2 * cm))
+
+    story.append(Spacer(1, 1 * cm))
     story.append(Paragraph(_("""
 Dear %s, this document contains the credentials you will need
 to connect to Modoboa. Learn the content and destroy
 the document as soon as possible.
 """) % account.fullname, styles["Normal"]))
-    url = "http(s)://{}{}".format(
-        Site.objects.get_current().domain, settings.LOGIN_URL)
+
+    story.append(Spacer(1, 0.2 * cm))
+    story.append(Paragraph(_("Webinterface:"), styles["h3"]))
+
+    url = "https://{}".format(
+        Site.objects.get_current().domain)
```

Comment:
Not always because modoboa can be configured on a sub url, not on the root one. (ie example.com/modoboa/)